packages/fetchai/skills/fipa_dummy_buyer/dialogues.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""
+This module contains the classes required for dialogue management.
+
+- FipaDialogue: The dialogue class maintains state of a dialogue of type fipa and manages it.
+- FipaDialogues: The dialogues class keeps track of all dialogues of type fipa.
+"""
+
+from typing import Any, Optional, Type
+
+from aea.common import Address
+from aea.exceptions import AEAEnforceError, enforce
+from aea.helpers.transaction.base import Terms
+from aea.protocols.base import Message
+from aea.protocols.dialogue.base import Dialogue as BaseDialogue
+from aea.protocols.dialogue.base import DialogueLabel as BaseDialogueLabel
+from aea.skills.base import Model
+
+from packages.fetchai.protocols.fipa.dialogues import FipaDialogue as BaseFipaDialogue
+from packages.fetchai.protocols.fipa.dialogues import FipaDialogues as BaseFipaDialogues
+from packages.fetchai.protocols.fipa.message import FipaMessage
+
+
+class FipaDialogue(BaseFipaDialogue):
```

Comment:
just do `FipaDialogue = BaseFipaDialogue` as you're not using Terms!

Reply:
done